flash/image/keypoint_detection/data.py

```diff
@@ -11,17 +11,16 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from functools import partial
-from typing import Any, Callable, Dict, List, Optional, Tuple, Type, Union
+from typing import Any, Dict, List, Optional, Type
 
-from flash.core.data.data_module import DataModule
 from flash.core.data.data_pipeline import DataPipelineState
-from flash.core.data.io.input import InputFormat
-from flash.core.data.io.input_transform import InputTransform
+from flash.core.data.io.input_base import Input
+from flash.core.data.new_data_module import DataModule
 from flash.core.integrations.icevision.data import IceVisionInput
-from flash.core.integrations.icevision.transforms import default_transforms
+from flash.core.integrations.icevision.transforms import IceVisionInputTransform as KeypointDetectionInputTransform
```

Comment:
That looks a bit strange, I think it might be simpler to just have:
```suggestion
from flash.core.integrations.icevision.transforms import IceVisionInputTransform
```